src/main/java/org/shipkit/gh/release/GitHubReleaseTask.java

```diff
@@ -80,12 +81,22 @@ public void setWriteToken(String writeToken) {
         this.writeToken = writeToken;
     }
 
+    @Input
+    public String getNewTagRevision() {
+        return newTagRevision;
+    }
+
+    public void setNewTagRevision(String newTagRevision) {
```

Comment:
Javadoc ;-)

Reply:
Thanks for review! I have just sent commits according to your suggestions.

Reply:
Great docs, thanks!